grails-app/conf/UrlMappings.groovy

```diff
@@ -82,7 +82,7 @@ class UrlMappings {
             action = [GET: "paymentTermOptions"]
         }
 
-        "/api/users" {
+        "/api/usersOptions" {
```

Comment:
Why this change? You should not change an existing endpoint like that.

Reply:
In my opinion, this should have been done like that in the first place.
This is an endpoint from `selectOptionsApiController` and looking at previous "...Options" endpoints, this is the approach we went with for these kinds of endpoints.

Also, this endpoint is only used in debounce fetch for user-select which data was transformed on the frontend.